src/ControlSystem/Tags.hpp

```diff
@@ -184,4 +170,66 @@ struct MeasurementTimescales : db::SimpleTag {
   }
 };
 }  // namespace Tags
+
+/// \ingroup ControlSystemGroup
+/// Place to hold all options for a single control system
+namespace OptionHolder {
```

Comment:
Is there anything else that will go in this namespace?  If not, let's use it as the class name instead.

Reply:
No nothing else is in this namespace. I'll use it as the class name instead.